components/Order/basic_usage.rst

```diff
@@ -0,0 +1,251 @@
+    echo $order->getTotal(); // 1650.
+
+The main order total is a sum of the previously mentioned values.
+You can access the order total value using the ``->getTotal()`` method.
+
+Recalculation of totals can happen by calling ``->calculateTotal()`` method, using the simplest math. It will also update the item totals.
+
+Items Management
+~~~~~~~~~~~~~~~~
+
+The collection of items (Implementing the ``Doctrine\Common\Collections\Collection`` interface) can be obtained using the ``->getItems()``.
+To add or remove items, you can simply use the ``addItem`` and ``removeItem`` methods.
+
+.. code-block:: php
+
+    <?php
+
+    use Sylius\Component\Order\Model\Order;
+    use Sylius\Component\Order\Model\OrderItem;
+
+    $order = new Order();
+
+    $item1 = new OrderItem();
+    $item1->setName('Super cool product');
+    $item1->setUnitPrice(1999); // 19.99!
+    $item1->setQuantity(2);
+
+    $item2 = new OrderItem();
+    $item2->setName('Interesting t-shirt');
+    $item2->setUnitPrice(2549); // 25.49!
+
+    $order->addItem($item1);
+    $order->addItem($item2);
+    $order->removeItem($item1);
+
+Order adjustment
+~~~~~~~~~~~~~~~~
+
+The order holds adjustments.
+
+.. code-block:: php
+
+    <?php
+
+    use Sylius\Component\Order\Model\Order;
+    use Sylius\Component\Order\Model\OrderItem;
+    use Sylius\Component\Order\Model\Adjustment;
+
+    $order = new Order();
+    $item = new OrderItem();
+    $adjustment = new Adjustment();
+
+    $adjustment->setAmount(1200);
+    $adjustment->setLabel('tax');
+
+    $item->setOrder($order);
+    $item->setQuantity(2);
+    $item->setUnitPrice(2000);
+
+    $order->addAdjustment($adjustment);
+    $order->addItem($item);
+    $order->calculateTotal();
+
+    $order->getTotal(); //5200
+
+Order Item
+----------
+
+An order item model has only the **id** property as identifier and it has the order reference, accessible via ``->getOrder()`` method.
+
+Order Item totals
+~~~~~~~~~~~~~~~~~
+
+Just like for the order, the total is available via the same method, but the unit price is accessible using the ``->getUnitPrice()``
+Each item also can calculate its total, using the quantity (``->getQuantity()``) and the unit price.
+
+.. code-block:: php
+
+    <?php
+
+    use Sylius\Component\Order\Model\OrderItem;
+
+    $item = new OrderItem();
+    $item->setUnitPrice(2000);
+    $item->setQuantity(4);
+    $item->calculateTotal();
+
+    $item->getTotal(); // 8000.
+
+Order Item adjustment
+~~~~~~~~~~~~~~~~~~~~~
+
+An OrderItem can also hold adjustments.
+
+.. code-block:: php
+
+    <?php
+
+    use Sylius\Component\Order\Model\OrderItem;
+    use Sylius\Component\Order\Model\Adjustment;
+
+    $adjustment = new Adjustment();
+    $adjustment->setAmount(1200);
+    $adjustment->setLabel('tax');
+
+    $item = new OrderItem();
+    $item->addAdjustment($adjustment);
+    $item->setUnitPrice(2000);
+    $item->setQuantity(2);
+    $item->calculateTotal();
+
+    $item->getTotal(); //5200
```

Comment:
I think we should keep this part, cause it was not mentioned anywhere before that order items can have adjustments.